Architect-Coders/MU8
# Issue #39: Save Comic id on Characters Entity

**What's in:**

- Created strategy to save the comics id on the Characters entity.

**Testing Done:**

- On Emulator: Pixel 3a.

**How it looks:** 

- Same as before, no visual changes done.

**How to test:**

- Run the app and reach the home screen.
- Before entering to the Characters screen put a breakpoint in line `57` of method `updateUi` inside `CharactersActivity` class as shown on the image below.

    ![image](https://user-images.githubusercontent.com/3532556/71927331-3d687280-3163-11ea-9d9c-dec838c9a510.png)

- Now enter to the Characters screen and upon reaching the breakpoint, check that the characters inside the model have now a list of ids for the comics the hero appears.
- Check that the functionality of the app remains the same for this and the other categories.

**Signed-off-by:**

- Diego Recalde <difereto89@gmail.com>
